### ISSUE TITLE
'feature' not in batch.keys() in loss_superpixels_average

### ISSUE THREAD

**Issue by synsin0** [sbysbysbys/UOV#1]
Hello, I preprocess with SAN and find that the script do not save xxx_lidarseg.npz, features=[] actually, and the collate function does not include 'feature' key to batch dict. How to generate features for superpoints?

**Comment by sbysbysbys** [comment]
Sorry, I may have accidentally deleted pretrain.dataloader_nuscenes_ov, I've added it back now. Please let me know if you encounter any issues.

**Comment by synsin0** [comment]
Thanks! I have another question with task settings, does the model learn open-vocabulary semantics on pretraining stage or annotation-free stage? Is pretrain is only for backbone warmup?

**Comment by sbysbysbys** [comment]
It learns semantics in both stages, with the first stage focusing on scene understanding and the second stage on scene perception. So pre-training not only warmup parameters but also endows our models with scene understanding and transfer learning capabilities.

**Comment by synsin0** [comment]
Great Work! Now I can start the pretrain stage and will reach you if meet further issues. Thanks a lot.
